src/index.html

```diff
@@ -115,7 +115,7 @@ <h3 class="section-heading">Les Rencontres JS nantaises</h3>
                         <span class="date">19/11/2015</span>
                         <span class="place">@ iAdvize</span>
                         <span class="sponsor"><i class="fa fa-hand-peace-o"></i> Seyos</span>
-                        <img class="u-max-full-width" src="images/meetup/flux.svg" alt="Meetup 14" title="Flux as seen in NantesJS meetup 14">
```

Comment:
En fait ils n'ont aucune pertinence, ni plus-value :
- c'est un mélange de FR et EN
- aucune information réelle n'y est fournie
- les title des derniers events étaient tous identiques et incomplets (" as seen in NantesJS meetup 15")